.github/workflows/build_packages.yaml

```diff
@@ -386,7 +386,22 @@ jobs:
         username: ${{ secrets.DOCKER_HUB_USER }}
         password: ${{ secrets.DOCKER_HUB_TOKEN }}
     - uses: docker/build-push-action@v2
-      if: github.event_name == 'release'
+      if: github.event_name == 'release' && github.event.release.prerelease
```

Comment:
does `github.event.release.prerelease` mean it *is* a prerelease or it *is not* a prerelease ?